Lib/test/test_fractions.py

```diff
@@ -1,5 +1,3 @@
-"""Tests for Lib/fractions.py."""
-
```

Comment:
This is an unrelated change.

Reply:
Fixed in [b585219](https://github.com/python/cpython/pull/119242/commits/b585219ac44c740baaec336ef6b07a17989f050a)